pyccel/ast/literals.py

```diff
@@ -0,0 +1,133 @@
+""" This module contains all literal types
+"""
+from sympy               import Integer as sp_Integer
+from sympy               import Float as sp_Float
+from sympy.logic.boolalg import BooleanTrue as sp_BooleanTrue, BooleanFalse as sp_BooleanFalse
+from sympy.core.expr     import Expr
+
+from .basic              import PyccelAstNode, Basic
+from .datatypes          import (NativeInteger, NativeBool, NativeReal,
+                                  NativeComplex, NativeString, default_precision)
+
+__all__ = (
+    'LiteralBooleanTrue',
+    'LiteralBooleanFalse',
+    'LiteralInteger',
+    'LiteralFloat',
+    'LiteralComplex',
+    'LiteralImaginaryUnit',
+    'LiteralString',
+    'get_default_literal_value'
+)
+
+#------------------------------------------------------------------------------
+class LiteralBooleanTrue(sp_BooleanTrue, PyccelAstNode):
```

Comment:
Is this inheritance useful? Should we use this PR to also remove sympy from this file as it is not doing anything?

Reply:
We need this as long as we are using `sympy.Function` as sympy "sympify"s arguments passed to the generated function. There does not seem to be a way to prevent this

Reply:
We'll create an Epic to tackle this.